polars/io.ts

```diff
@@ -476,33 +476,56 @@ interface RowCount {
 }
 
 interface ScanParquetOptions {
-  columns?: string[] | number[];
-  numRows?: number;
+  nRows?: number;
+  cache?: boolean;
   parallel?: "auto" | "columns" | "row_groups" | "none";
   rowCount?: RowCount;
-  cache?: boolean;
   rechunk?: boolean;
-  hive_partitioning?: boolean;
+  rowCountName?: string;
+  rowCountOffset?: number;
```

Comment:
We already have `rowCount` object that users should use. No need to have it here too. 

`scanParquet({rowCount: {name: 'name', offset: 0}, ...})`